manage.py

```diff
@@ -43,19 +48,43 @@ def test(all=False):
 
 @manager.command
 def scrape():
-    """Start an infinte loop to scrape articles."""
-    scraper = Scraper()
+    """Start an infinte loop to scrape & categorize articles."""
     iterations = 0
+    categorizer = Categorizer()
 
     while True:
+        logging.info("Starting scrape...")
         if (iterations % ITERATIONS_BEFORE_RESET) == 0:
-            scraper.reset_title_categories()
+            categorizer.reset_title_categories()
 
-        scraper.scrape_articles()
+        scrape_articles(categorizer)
 
         iterations += 1
+        logging.info("Finished scrape. Zzz...")
         time.sleep(SCRAPE_INTERVAL)
 
+@manager.command
+def seed():
+    """Add seed data to the database"""
+    """Required: Need to delete database & run db upgrade first"""
+    # One user
+    user = User(email="admin@sourcemash.com", password="password")
+    db.session.add(user)
+    db.session.commit()
+
+    # One feed
+    feed = Feed(id=1, title='TechCrunch > Startups', 
```

Comment:
Changed NYTFeedFactory to TechcrunchFeedFactory, but don't want to import factories from tests here, so we'll continue to do it manually.